apps/src/templates/sectionAssessments/StudentsAssessmentsMCTable.jsx

```diff
@@ -96,6 +97,12 @@ class StudentsAssessmentsMCTable extends Component {
     });
   };
 
+  nameCellFormatter = (name, {rowData}) => {
+    return (
+      <a href={rowData.url}>{name}</a>
```

Comment:
do we need to add any styles to the link? i remember the other student names being blue, but i don't remember if they were also links

Reply:
Good question.  Styles were added to the link but the uploaded screen shot shows the students' names that have been clicked.